hack/e2e-test/build-clusters.sh

```diff
@@ -38,9 +38,9 @@ util::create_cluster "${HOST_CLUSTER_NAME}" "${MAIN_KUBECONFIG}" "${KIND_VERSION
 util::create_cluster "${MEMBER_CLUSTER_NAME}" "${MEMBER_CLUSTER_KUBECONFIG}" "${KIND_VERSION}" "${TEMP_PATH}" "${TEMP_PATH}"/member1.yaml
 
 util::check_clusters_ready "${MAIN_KUBECONFIG}" "${HOST_CLUSTER_NAME}"
-sleep 5s
+sleep 5
```

Comment:
I don't think sleep is needed here, at least not a hardcoded sleep, but currently removing it will cause the test environment build to fail. Removing `s` is to ensure it is compatible with mac system.